### ISSUE TITLE
lib.php referencing quickmail plugin

### ISSUE THREAD

**Issue by bel-omahoney** [roelmann/moodle-block_course_contacts#16]
Hi, all of the functions in the lib.php file seem to be referencing the quickmail plugin and the quickmail db tables. None of these functions are referenced in the rest of the plugin and quickmail is not listed as a dependency for this plugin. I have installed block_course_contacts without this file and all works as expected. Is quickmail required for this plugin? Is the lib.php file required? Thanks

**Comment by roelmann** [comment]
TBH my updates so far have been limited to ensure the block works on the respective updated moodles. I have never done the refactoring I originally wanted to do when I took on the maintenance of the block.
By all means test it by removing the file, and feed back (would be much appreciated!) 
You will always be able to restore the block from here if it crashes, but as you say, I am aware it (historically) references quickmail, but appears to work fine without it installed.

**Comment by bel-omahoney** [comment]
Hi roelmann, I tested this block without the lib.php file and all works fine, this file must be in there from the previous version with the quickmail dependency but is no longer needed. My version of moodle does not have the quickmail plugin, so it must not be needed. As stated, every function in there is for use with quickmail and not referenced anywhere else in the plugin. Just for your info, I also deleted the email.php, email_form.php and settings.php (which only has settings relevant to simple email) because the use of simple email was commented out and also not used anywhere, and all works fine still. I am aware that this is being left in the plugin because it is intended to be fixed (or was?) but the info might be helpful. Thanks
